core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/ConfigDocItemScanner.java

```diff
@@ -49,7 +50,7 @@ public ConfigDocItemScanner() {
      */
     public void addConfigGroups(TypeElement configGroup) {
         String configGroupName = configGroup.getQualifiedName().toString();
-        if (configGroupName.startsWith(IO_QUARKUS_TEST_EXTENSION_PACKAGE)) {
+        if (!configGroupName.startsWith(IO_QUARKUS_TEST_EXTENSION_PACKAGE)) {
```

Comment:
Is this change and the one below intentional?

Reply:
Very good catch. I knew I was going to forget about this! (it was to test stuff).